templates/azure/pro.html

```diff
@@ -5,12 +5,20 @@
 {% block meta_copydoc %}https://docs.google.com/document/d/1WaQ5n3GxwZzbX0T6rLiRNejFvxIPIHd5V5XgdAwKmSw/edit{% endblock meta_copydoc %}
 
 {% block content %}
-<section class="p-strip--suru-topped u-no-padding--bottom">
+<section class="p-strip--suru-topped is-bordered">
   <div class="row u-equal-height">
     <div class="col-7 col-medium-4">
       <h1>Ubuntu Pro for Azure</h1>
       <p class="p-heading--3">For production. For professionals.</p>
       <p>Ubuntu Pro for Azure is a premium image delivering the most comprehensive open source security and Azure compliance. Ubuntu Pro is suitable for small to large-scale Linux enterprise operations offering pay-as-you-go billing on your existing Azure invoice.</p>
+      <div class="u-sv2">
+        <h3 class="p-heading--4" style="display:inline;padding-right:0.8rem;">Ubuntu Pro 22.04 LTS</h3>
+        <button class="p-chip--positive is-inline">
```

Comment:
Mouse pointer appears when hovering over the button but it doesn't call any action.. Could we disable changing mouse pointer?